ui/src/environments/environment.prod.ts

```diff
@@ -19,7 +19,7 @@ export const environment: Environment = {
   trainingUrl: 'https://aou.nnlm.gov',
   inactivityTimeoutSeconds: 30 * 60,
   inactivityWarningBeforeSeconds: 5 * 60,
-  enableCaptcha: false,
+  enableCaptcha: true,
```

Comment:
Existing issue, but it looks like this is redefining a value that's set in the server-side config. Probably better to pass this from the server. Fine as a follow-up if this is large, or maybe we could just wait to delete this, depending on how long we think the flag will stay around.

Reply:
hmm since it will be part of next weeks release, i can have a clean up PR ready for next week?

Reply:
Sorry, forgot to reply ehre - yes that's fine